File: para.java
public class Add{
 
    public static void main(String[] args) {
        // Check if two arguments are passed
        if (args.length != 2) {
            System.out.println("Please provide two integer arguments.");
            return;
        }
 
        // Parse the command-line arguments
        int first = Integer.parseInt(args[0]);
        int second = Integer.parseInt(args[1]);
 
        // Add the two numbers
        int sum = first + second;
 
        // Print the result
        System.out.println(first + " + " + second + " = " + sum);
    }
}

File: para.py
import sys

# Get the parameters passed from Jenkins
num1 = int(sys.argv[1])
num2 = int(sys.argv[2])

# Perform addition
result = num1 + num2

# Print the result
print(f"The sum of {num1} and {num2} is {result}")


python first.py %first% %second%
